homeassistant/components/meater/__init__.py

```diff
@@ -51,8 +51,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         await meater_api.authenticate(
             entry.data[CONF_USERNAME], entry.data[CONF_PASSWORD]
         )
-    except Exception:
-        _LOGGER.error("Unable to authenticate with the Meater API")
+    except Exception as err:
+        _LOGGER.error(f"Unable to authenticate with the Meater API: {err}")
```

Comment:
```suggestion
        _LOGGER.error("Unable to authenticate with the Meater API: %s", err)
```

For logging we don't do f-strings, usually.

Reply:
Thanks, I think I actually knew that but I must've been in auto-pilot when I wrote that line.